networking/src/NTP/Client.hs

```diff
@@ -125,6 +124,14 @@ handleCollectedResponses cli = do
   where
     handleE = logError . sformat ("ntpMeanSelection: "%shown)
 
+gatheredAllResponses :: NtpClient m -> STM Bool
```

Comment:
If we want this to be read as a question, this variable should be called `allResponsesGathered` :)

Reply:
Yeah, it looks better 👍